main.cpp

```diff
@@ -60,12 +60,41 @@ send me a DM to check your pull request
 //2)
 void variableDeclarations()
 {
-    //example:
-    int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
-    
-    
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int eggs = 0;
+    int chocolateChips = 1;
+    int butterSticks = 1;
+
+    float flour = 1.f;
+    float cinnamon = 0.25f;
+    float sugar = 1.f;
+
+    double cookTime = 60.5;
+    double ovenTemperature = 400.5;
+    double cupsOfMilk = 1.75;
+
+    bool ovenOn = true;
+    bool mealReadyToCook = false;
+    bool mealCooked = false;
+
+    char stepOne = 'a';
+    char stepTwo = 'b';
+    char stepThree = 'c';
+
+    ignoreUnused(eggs);
+    ignoreUnused(chocolateChips);
+    ignoreUnused(butterSticks);
+    ignoreUnused(flour);
+    ignoreUnused(cinnamon);
+    ignoreUnused(sugar);
+    ignoreUnused(cookTime);
+    ignoreUnused(ovenTemperature);
+    ignoreUnused(cupsOfMilk);
+    ignoreUnused(ovenOn);
+    ignoreUnused(mealReadyToCook);
+    ignoreUnused(mealCooked);
+    ignoreUnused(stepOne);
+    ignoreUnused(stepTwo);
+    ignoreUnused(stepThree);
```

Comment:
Can you pass all of these to a single call of `ignoreUnused();` 

- [ ] pass all local variables to a single `ignoreUnused` call